src/components/common/Extensions/NotificationBanner/NotificationBanner.tsx

```diff
@@ -56,6 +58,7 @@ const NotificationBanner: FC<PropsWithChildren<NotificationBannerProps>> = ({
           <div
             className={clsx('text-sm max-w-[50rem] mt-1.5', {
               'text-negative-400': status === 'error',
+              'text-warning-400 font-semibold': status === 'warning',
```

Comment:
I think you can add `font-semibold` to children that you are passing to this component